docker-compose.yaml

```
@@ -0,0 +1,76 @@
+services:
+  mariadb:
+    image: mariadb:11.5.2
```

if you add the full latest to the mariadb, this has its pros and cons, on one hand its a specific version, but on the other, every time version changes we have to change it, maybe this might be better
```suggestion
    image: mariadb:11
```